### PR TITLE
Improve linter

### PR DESCRIPTION
**Pull request by viacheslav-iskenderov**

### CODE REVIEW

File: client/src/index.jsx [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 import { createRoot } from 'react-dom/client';
 import { Provider } from 'react-redux';
 import { BrowserRouter as Router } from 'react-router-dom';
-import { App } from 'pages/app/app';
-import { store } from 'packages/store/store';
-
-import 'assets/css/styles.scss';
 
-const root = createRoot(document.getElementById('root'));
+const root = createRoot(document.querySelector('#root'));
```
Comment (v0ldemar01) [inline client/src/index.jsx:10]: Duplicating of document.querySelector('#root'))

File: commitlint.config.cjs [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+const COMMIT_MESSAGE_MATCH_RULE_MESSAGE = `commit message doesn't match format requirements
+Commit message must have one of the following formats:
+  - <project-prefix>-<issue-number>: <modifier> <description>
+  - <environment>: <modifier> <description>
+Where:
+  - <project-prefix>: ${ProjectPrefix.APP}
+  - <modifier>: ${COMMIT_MODIFIERS.join(', ')}
+  - <environment>: ${ProjectPrefix.ENVIRONMENTS.join(', ')}
+Examples:
+  - BSA-5: + ui/ux lecture
```
Comment (v0ldemar01) [inline commitlint.config.cjs]: Let's add prefix from this project, from ProjectPrefix

File: dangerfile.js [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+  }
+};
+
+const checkMilestone = () => {
+  const hasMilestone = Boolean(pr.milestone);
+
+  if (!hasMilestone) {
+    fail('This pull request should have a milestone.');
+  }
+};
```
Comment (v0ldemar01) [inline dangerfile.js]: I'm not sure that we will have these ones here

File: readme.md [hunk 4]
```diff
@@ -180,13 +186,76 @@ Shared package contains code that is used for both frontend and backend.
...
+```
+
+##### Types
+
+- task
+- fix
+
+##### Examples
+
+- `task/sl-5-add-clinician-dashboard`
```
Comment (v0ldemar01) [inline readme.md]: It's reference to simpleland

File: server/src/data/seeds/fill-database.js [hunk 5]
```diff
@@ -40,19 +43,19 @@ export async function seed(knex) {
...
-        [ColumnName.IMAGE_ID]: userImages[idx] ? userImages[idx].id : null
+        [ColumnName.IMAGE_ID]: userImages[index] ? userImages[index].id : null
       }));
       const users = await trx(TableName.USERS)
         .insert(usersMappedSeed)
         .returning('*');
 
       // Add posts.
-      const postsMappedSeed = postsSeed.map((post, idx) => ({
+      const postsMappedSeed = postsSeed.map((post, index) => ({
```
Comment (v0ldemar01) [inline server/src/data/seeds/fill-database.js:55]: Just for future, just create a new task for code refactor)

**Review by v0ldemar01 — CHANGES_REQUESTED** [review]

**Review by v0ldemar01 — CHANGES_REQUESTED** [review]
And please follow the commit flow(use not production but task prefix))

File: server/src/data/seeds/fill-database.js (hunk 5 above)
Comment (viacheslav-iskenderov) [reply server/src/data/seeds/fill-database.js:55]: It's a part of the linter rule. It forces to use full word instead of shortcuts

**Pull request merged**